pkg/registry/thirdpartyresourcedata/codec.go

```diff
@@ -41,14 +41,43 @@ type thirdPartyResourceDataMapper struct {
 
 var _ meta.RESTMapper = &thirdPartyResourceDataMapper{}
 
+func (t *thirdPartyResourceDataMapper) getResource() unversioned.GroupVersionResource {
+	return unversioned.GroupVersionResource{Group: t.group, Version: t.version, Resource: t.kind + "s"}
+}
+
+func (t *thirdPartyResourceDataMapper) getKind() unversioned.GroupVersionKind {
+	return unversioned.GroupVersionKind{Group: t.group, Version: t.version, Kind: t.kind}
+}
+
 func (t *thirdPartyResourceDataMapper) isThirdPartyResource(resource string) bool {
 	plural, _ := meta.KindToResource(t.kind, false)
```

Comment:
Need to update this method to account for the signature change to `KindToResource`